### DISCUSSION TITLE
Welcome to TrabalhoPDS Discussions!

### DISCUSSION THREAD

**Discussion by LayaneGP** [Trabalho-PDS/TrabalhoPDS#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LayaneGP** [comment]
@sarahComH @LcsDeveloper 
Coloquei aquela mensagem de início porque é padrão kkkkkkk tava testando. Enfim... quero ver com vcs duas coisas:
1- @sarahComH , com as últimas modificações que vc fez, não serão necessárias alterações nas classes relacionadas a usuários? 
2- para ambos: como falei com vcs, vai ser difícil eu ajudar muita coisa daqui pra frente, mas não posso ficar sem fazer nada, então to pensando em fazer makefile, docfile  e readme (vou precisar de ajuda, mas vou tentar sozinha primeiro tudo que nao precisar de discussao pra ser decidido). 
Tendo isso em vista, seria interessante vcs comentarem as funções conforme queiram que elas sejam explicadas no docfile (ou pelo menos com uma ideia da funcionalidade, e eu desenvolvo melhor). Vai facilitar pra eu nao precisar perguntar vcs o tempo todo se to entendendo certo.

por enquanto é isso, o que acham (sobre todos os pontos)?

**Reply by sarahComH** [reply]
1 - Vão ser necessárias... eu achei q vc ia acabar o gerenciar usuário p gnt juntar as classes (sem ele meio que nn teria como). Nesse caso, agnt ia discutir, ver oq dá p manter e o q nn dá, adicionaria coisas no repositorio usuarios tbm e chegaria em um alinhamento. Igual eu comentei com vcs lá no grupo, a main foi só uma demonstração das classes funcionando e agnt podendo interagir com o projeto antes de ele estar 100% pronto.
2 - As classes de usuário que estão aqq no projeto já estão prontas? 
Sobre comentar as funções, comento ss. Mas acho q a prioridade agora é definir se todas as classes estão certas e tals.

**Reply by lucasrpera** [reply]
1- Eu fiz algumas modicações em algumas classes antigas do usuário pra conseguir implementar com a interface por enquanto, mas tá funfando, qualquer coisa a gente pode deixar assim por enquanto se precisar.
2- Eu concordo sim! O makefile tem aquela minha base, eu ia fazer o request mas me enrolei aqui, vou fazer o quanto antes caso queira usar como base. Sobre o que a Sarah apontou eu também concordo, corrigir o que ainda não estiver 100%, tem mais prioridade, mas me coloco a disposição para olhar isso nas classes que estavam com você, para que foque nas demais coisas que faltam.

**Reply by LayaneGP** [reply]
beleza, lucas, fechou então. Quando vc fizer o pull do makefile eu dou uma olhada. Até lá, a partir de quarta 13h (pós minha prova de cálculo), vou ficar por conta de aprender como fazer isso e o restante dos documentos que citei.

**Reply by lucasrpera** [reply]
Realizei agora o pull!!

**Comment by LayaneGP** [comment]
> 1 - Vão ser necessárias... eu achei q vc ia acabar o gerenciar usuário p gnt juntar as classes (sem ele meio que nn teria como). Nesse caso, agnt ia discutir, ver oq dá p manter e o q nn dá, adicionaria coisas no repositorio usuarios tbm e chegaria em um alinhamento. Igual eu comentei com vcs lá no grupo, a main foi só uma demonstração das classes funcionando e agnt podendo interagir com o projeto antes de ele estar 100% pronto. 2 - As classes de usuário que estão aqq no projeto já estão prontas? Sobre comentar as funções, comento ss. Mas acho q a prioridade agora é definir se todas as classes estão certas e tals.

1- então, como falei com vcs no grupo, acho bem difícil que eu consiga, to numa situação terrível em outras matérias, e pds ate que da pra passar, mas como minha prioridade é nao repetir, isso nao vai rolar se eu focar mais nesse trabalho desenvolvendo as classes. 
2- até onde me lembro, as classes de usuario ainda têm as funções de modificar coisas que decidimos nao mais modificar. A classe dos estudantes não está terminada também, fiquei sem saber o que colocar
Ok, concordo que a prioridade é essa. Se for possível ir comentando durante as atualizações creio que o tempo gasto vai ser menor, e vai fazer as duas coisas ao mesmo tempo. Mas se não der, não tem problema
